index.html

```diff
@@ -1,21 +1,114 @@
+          <a href="#" class="Features">Features</a>
+          <a href="about.html" class="About">About</a>
+          <a href="#" class="Contact">Contact</a>
+        </nav>
+          <a href="index.html" class="index">
+            <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+          </a>
+    </header>
+
+      </div>
+
+    <section class="top-main">
+
+      <div class="top-text">
+        <h1>
+          Innovation
+          <br> On
+          <br> Demand
+        </h1>
+        <a href="#">
+          <button class="btn default">Get Started</button>
+        </a>
+      </div>
+      <div class="top-img">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>
+
+
+    <br>
+    <div class="first-main">
+
+      <section class="features">
+        <h2>Features</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+
+      <section class="about">
+        <h2>About</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+    </div>
+
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="second-main">
+      <section class="services">
+        <h2>Services</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+
+      <section class="products">
+        <h2>Product</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+
+      <section class="vision">
+        <h2>Vision</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse
+          eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </section>
+    </div>
+
+    <footer>
+      <section class="contact">
+        <h2>Contact</h2>
+        <address>
+          123 Way 456 Street
+          <br> Somewhere, USA
+          <br>
+          <br>
+          <a href="tel:1-888-888-8888" class="tel"> (888)-888-8888</a>
+          <br>
+          <a href="mailto:sales@greatidea.io" class="mailto">sales@greatidea.io </a>
+        </address>
+      </section>
+      <br>
+      <h3> Copyright Great Idea! 2018</h3>
+    </footer>
+  </div>
 </body>
```

Comment:
Your html is very uniform and clean.  Great job with this Stefan because code must be readable for others, not just you.